client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -18,31 +18,70 @@ import { updateTeam } from '../../actions/team';
 const ProfileInfoEdit = () => {
   const dispatch = useDispatch();
 
-  const { teamId, teamName, orgName, email } = useSelector(
+  const { teamId, teamName, orgName, email,password, confirmedPassword } = useSelector(
```

Comment:
I don't think we ever stored the password and confirmedPasswordfrom the redux state, so probably does not need to include here.